packages/moonstone/Panels/Header.module.less

```diff
@@ -84,7 +84,7 @@
 		}
 
 		.titlesCell {
-			margin-bottom: 9px;
+			margin-bottom: 15px;
```

Comment:
Should we be changing this for a standard header, too?  Is this something we can calculate from some other value?

Reply:
It could technically be calculated from other values but the equation is unintuitive and harder to maintain than just this value.

Oops! Nope, that shouldn't be affecting standard, just dense. I'll update this to only affect dense.